website/pages/blog/introducing-the-tailscale-source-plugin.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Introducing The Tailscale Source Plugin
+tag: announcement
+date: 2022/12/14
+description: >-
+  CloudQuery is an open-source, extensible framework that gives you a
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Tailscale source plugin.
+author: alex
+--- 
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL.
```

Comment:
Similar to https://github.com/cloudquery/cloudquery/pull/5643#discussion_r1048621581 can we drop the SQL as we have non SQL destinations and use the tagline from the homepage:
```
CloudQuery is an open source high performance data integration platform designed for security and infrastructure teams.
```